docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -269,6 +269,51 @@ The `$reindex` operation can be invoked via an HTTP(s) POST to `[base]/$reindex`
 |----|----|-----------|
 |`tstamp`|string|Reindex any resource not previously reindexed before this timestamp. Format as a date YYYY-MM-DD or time YYYY-MM-DDTHH:MM:DDZ.|
 |`resourceCount`|integer|The maximum number of resources to reindex in this call. If this number is too large, the processing time might exceed the transaction timeout and fail.|
+|`resourceLogicalId`|string|The ResourceType or the ResourceType/Logical id for targetted reindexing|
```

Comment:
Good documentation add.  A related thought on the design of this operation is that it might be more intuitive to support this capability by exposing `$reindex` at both the type and the index level.
This way, we wouldn't really need this parameter at all.

`POST [base]/$reindex` would reindex resources of any type
`POST [base]/Patient/$reindex` would reindex Patient resources
`POST [base]/Patient/123/$reindex` would reindex just Patient/123

Probably a better thought for #2195 ...

Reply:
I think it's an easy addition that I can add

Reply:
I'd like to get this in a follow on PR.

Reply:
Support $reindex operation supports resourceLogicalId on Type and Instance #2503